ui-tests/src/test/resources/features/connections/oauth.feature

```diff
@@ -1,79 +1,31 @@
-      | Gmail      | Test-Gmail-connection      |
-      | Salesforce | Test-Salesforce-connection |
-      | Twitter    | Test-Twitter-connection    |
+      | SAP Concur      | Test-Concur-connection          |
+      | Gmail           | Test-Gmail-connection           |
+      | Salesforce      | Test-Salesforce-connection      |
+      | Google Sheets   | Test-Google-Sheets-connection   |
+      | Google Calendar | Test-Google-Calendar-connection |
+      | Twitter         | Test-Twitter-connection         |
 
-    # create integration
-    When navigate to the "Home" page
-    And click on the "Create Integration" button to create a new integration.
-    Then check visibility of visual integration editor
 
-    When check that position of connection to fill is "Start"
-    # select twitter connection
-    And select the "Test-Twitter-connection" connection
-    And select "Mention" integration action
-    Then check that position of connection to fill is "Finish"
 
-    # select salesforce connection
-    When select the "Test-Salesforce-connection" connection
-    And select "Create or update record" integration action
-    And select "Contact" from "sObjectName" dropdown
-    And click on the "Next" button
-    And select "TwitterScreenName" from "sObjectIdName" dropdown
-    And click on the "Done" button
+#TODO: Necessary OAuth scenarios: Gmail, Salesforce, Twitter, Google Calendar
+#TODO: because mentioned connectors do not use OAuth in their tests
 
-    # add data mapper step
-    When add integration step on position "0"
-    And select "Data Mapper" integration step
-    Then check visibility of data mapper ui
-
-    When create data mapper mappings
-      | user.screenName | TwitterScreenName__c |
-      | text            | Description          |
-      | user.name       | FirstName; LastName  |
-    And scroll "top" "right"
-    Then click on the "Done" button
-
-    # finish and save integration
-    When click on the "Save" button
-    And set integration name "Twitter to Salesforce oauth"
-    And publish integration
-    Then Integration "Twitter to Salesforce oauth" is present in integrations list
-    # wait for integration to get in active state
-    And wait until integration "Twitter to Salesforce oauth" gets into "Running" state
-
-    #there is a problem that integration is not listening instantly after publishing so delay is necessary - probably start-up time
-    When sleep for "15000" ms
-    Then check SF does not contain contact for tw account: "twitter_talky"
-
-    When tweet a message from twitter_talky to "Twitter Listener" with text "OAuth syndesis test"
-    # give salesforce time to create contact
-    And sleep for "5000" ms
-    Then validate contact for TW account: "twitter_talky" is present in SF with description: "OAuth syndesis test"
-    And clean application state
-    And clean SF contacts related to TW account: "twitter_talky"
-    And clean all tweets in twitter_talky account
```

Comment:
why you didn't keep the test and didn't create a new one for the validation?

Reply:
It was testing 2 OAuth connections at once so if one fails, the whole test fails. Also it uses twitter - the one whith sms troubles ^^ So I figured it would be best to get rid of it and create separate scenarios for each OAuth, there is a JIRA for that.